.github/workflows/docker-workflow.yml

```diff
@@ -0,0 +1,262 @@
+          LATEST: ${{ inputs.push_latest }}
+        run: |
+          PUB_DOCKER_IMAGE=""
+          DOCKER_IMAGES_TMP=${{ inputs.docker_registry }}/${{ inputs.docker_image }}:${{ inputs.version }}
+
+          if [ "${LATEST}" = "true" ]; then
+            DOCKER_IMAGES_TMP="$DOCKER_IMAGES_TMP,${{ inputs.docker_registry }}/${{ inputs.docker_image }}:latest"
+          fi
+
+          # If public push is needed
+          if [ "${PUBLIC_PUSH}" = "true" ]; then
+            PUB_DOCKER_IMAGE_TMP="${{ inputs.public_registry }}/${{ inputs.public_image }}:${{ inputs.version }}"
+            GITHUB_DOCKER_IMAGE_TMP="${{ inputs.github_registry }}/${{ inputs.public_image }}:${{ inputs.version }}"
+            DOCKER_IMAGES_TMP="$DOCKER_IMAGES_TMP,$PUB_DOCKER_IMAGE_TMP,$GITHUB_DOCKER_IMAGE_TMP"
+            #echo "::set-output name=PUB_DOCKER_IMAGE::$PUB_DOCKER_IMAGE"
+            echo "PUB_DOCKER_IMAGE=$PUB_DOCKER_IMAGE_TMP" >> $GITHUB_OUTPUT
+            echo "GITHUB_DOCKER_IMAGE=$GITHUB_DOCKER_IMAGE_TMP" >> $GITHUB_OUTPUT
+
+            if [ "${LATEST}" = "true" ]; then
+              DOCKER_IMAGES_TMP="$DOCKER_IMAGES_TMP,${{ inputs.public_registry }}/${{ inputs.public_image }}:latest,${{ inputs.github_registry }}/${{ inputs.public_image }}:latest"
+            fi
+          fi
+
+          #echo "::set-output name=DOCKER_IMAGES::$DOCKER_IMAGES"
+          echo "DOCKER_IMAGES=$DOCKER_IMAGES_TMP" >> $GITHUB_OUTPUT
+          echo -e "Docker images to publish:\n$DOCKER_IMAGES"
+
+      - name: Compute docker build arguments
+        id: get_build_args
+        run: |
+          BUILD_ARGS_STR_LIST="${{ inputs.build_args }}"
+          if [ -n "$BUILD_ARGS_STR_LIST" ]; then
+            #echo "::set-output name=build_args_str_list::$BUILD_ARGS_STR_LIST"
+            echo "build_args_str_list=$BUILD_ARGS_STR_LIST" >> $GITHUB_OUTPUT
+            echo -e "Docker build arguments:\n$BUILD_ARGS_STR_LIST"
+          fi
+
+      - name: Download packages to include in build
+        if: inputs.download_artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: packages
+          path: ${{ inputs.download_artifact_path }}
+
+      - name: Set up QEMU for multi platform
+        if: ${{ contains(inputs.platforms, 'arm') }}
+        uses: docker/setup-qemu-action@v1
+      # https://github.com/docker/setup-buildx-action
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build with args and push:${{ inputs.deploy }}
+        if: ${{ steps.get_build_args.outputs.build_args_str_list != '' }}
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: ${{ inputs.platforms }}
+          file: ${{ inputs.docker_file }}
+          push: ${{ inputs.deploy }}
+          tags: ${{ steps.get_image_names.outputs.DOCKER_IMAGES }}
+          labels: "org.opencontainers.image.source=https://github.com/${{ inputs.github_repo }}"
```

Comment:
also, https://github.com/MOV-AI/containers-mariana-test/pkgs/container/ce%2Fmariana-hello-world/111574779?tag=v1.1 reports a missing `description` label which we should add as well